### ISSUE TITLE
Determining back cover

### ISSUE THREAD

**Issue by broskoski** [evnbr/bindery#64]
Are there any strategies or suggestions for setting a back page cover (or the left side of the first spread)?

**Comment by evnbr** [comment]
the intended pattern here is just to include the cover as the last element in the content, and use a `FullBleedPage` rule. But it looks like i'm not exposing the api for "preferring" a side, which is used internally and inserts a blank page if necessary.

`FullBleedPage({ selector: ".your-div", prefer: "left" })`

Or were you trying to create the front and back cover as a single continuous element?

**Comment by broskoski** [comment]
@evnbr thanks for responding, yeah I was thinking full front and back as a single element but ended up just making a separate screen for it (as I wanted to make something that would calculate spine width for perfect binding, which I realize might be out of the scope for this lib).

**Comment by evnbr** [comment]
Yes, if you're printing with blurb or equivalent, i believe they're asking for a separate pdf for the cover+spine anyways and so wasn't considering that part in-scope.
